doc/man7/EVP_PKEY-EC.pod

```diff
@@ -14,9 +14,9 @@ The following Import/Export types are available for the built-in EC algorithm:
 
 =over 4
 
-=item "curve-name" (B<OSSL_PKEY_PARAM_EC_NAME>) <utf8 string>
+=item "group" (B<OSSL_PKEY_PARAM_GROUP_NAME>) <utf8 string>
```

Comment:
maybe use "group-name"

Reply:
I actually had that at one point and it broke some command line usage, so I went with "group" as the easier naming option (rather than fixing all the command line usage)